core/Azure.Mcp.Core/src/Services/Azure/Tenant/ITenantService.cs

```diff
@@ -102,4 +102,22 @@ public interface ITenantService
     Task<TokenCredential> GetTokenCredentialAsync(
         string? tenantId,
         CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Gets a new instance of <see cref="HttpClient"/> configured with the following options:
+    /// - Proxy
```

Comment:
There's a native list syntax for doc comments. Examples are in our various [`Program.cs`](https://github.com/microsoft/mcp/blob/18ed5f2cfd9373cd7775c93e5c10860cd63f90a7/servers/Azure.Mcp.Server/src/Program.cs#L141-L160) files for a big comment I added warning of dragons. [Official docs.](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list)